lib/DonorsList/hooks/useFetchDonors/useFetchDonors.js

```diff
@@ -0,0 +1,36 @@
+import { useMutation } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+
+import { VENDORS_API } from '../../../constants';
+import { batchRequest } from '../../../utils';
+
+const buildQueryByIds = (itemsChunk) => {
+  const query = itemsChunk
+    .map(chunkId => `id==${chunkId}`)
+    .join(' or ');
+
+  return query || '';
+};
+
+export const useFetchDonors = () => {
+  const ky = useOkapiKy();
+
+  const { isLoading, mutateAsync } = useMutation(
```

Comment:
Why don't you use `useQuery` for fetching by ids?

Reply:
The mutation will be passed as a prop inside the other components.

Reply:
Yeah, but you have the query key to invalidate the fn and you can use ids of donors for that purpose. I know, that there are examples of using `useMutation` to fetch the data, but probably `useQuery` will bring more benefits, like caching.